sock-merchant.ts

```diff
@@ -0,0 +1,12 @@
+export function sockMerchant(n: number, ar: number[]): number {
+  // Write your code here
+  const filteredArr = [...new Set(ar)];
+
+  let count = 0;
+
+  filteredArr.forEach((num) => {
```

Comment:
It works, but why didn't you use `reduce` which is more appropriate for your use case?